### PR TITLE
Add config file including all options

### PR DESCRIPTION
**Pull request by sosthene-nitrokey**
This PR adds an `p11nethsm.example.conf` including and explaining all the options. It also adds a unit test that checks all the  values of this file  once parsed.

### CODE REVIEW

**Comment by sosthene-nitrokey** [comment]
The test will conflict with https://github.com/Nitrokey/nethsm-pkcs11/pull/165

File: p11nethsm.example.conf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+enable_set_attribute_value: false
+
+# You can set the log file location here.
+# If no value is set the module will output to stderr.
+# If a value is set it will output to the file.
+log_file: /tmp/p11nethsm.log
+# Optional log level, acceptable  values are Trace, Debug, Info, Warn and Error
+log_level: Debug
+
+# Each "slot" represents a NetHSM server
```
Comment (robin-nitrokey) [inline p11nethsm.example.conf:14]: Actually, it’s a set of servers with the same users and keys (or at least a common subset), isn’t it?  Not sure if there is a preferred term for that setup.  Maybe “NetHSM cluster”?

**Review by robin-nitrokey — APPROVED** [review]

File: p11nethsm.example.conf (hunk 1 above)
Comment (nponsard) [reply p11nethsm.example.conf:14]: Yeah a slot represent a cluster of NetHSM that can be accessed with the same credentials.
https://docs.nitrokey.com/nethsm/pkcs11-setup#instances

Comment (sosthene-nitrokey) [reply p11nethsm.example.conf:14]: This was taken verbatim from [the docs](https://docs.nitrokey.com/nethsm/pkcs11-setup#configuration-file-format).
I Will fix it both here and there.

Comment (nponsard) [reply p11nethsm.example.conf]: Ah that's my bad. I should add that the slot name comes from PKCS#11 where it's like a card reader or HSM port.

Comment (sosthene-nitrokey) [reply p11nethsm.example.conf]: Once this is merged I will udpate it also in the docs.

**Pull request merged**